tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "include": [
     "src/**/*.ts",
     "test/**/*.ts",
-    "scripts/**/*.js",
+    "scripts/**/*.ts",
```

Comment:
I'm not sure if it's even needed, ts-node seems to be working without need for `tsc`

Reply:
This is useful for type-checking though. If we were to omit this, running `tsc` wouldn’t catch any type errors.